saltproc/depcode.py

```diff
@@ -36,13 +47,9 @@ def __init__(self,
                Name of depletion code.
            exec_path : str
                Path to depletion code executable.
-           template_inputfile_path : str
-               Path to depletion code input file template.
-           iter_inputfile : str
-               Name of depletion code input file for depletion code rerunning.
-           iter_matfile : str
-               Name of iterative, rewritable material file for depletion code
-               rerunning. This file is modified during  the simulation.
+           template_inputfiles_path: str or dict of str to str
```

Comment:
This description is also not clear. What str?

Reply:
I meant that it's not clear what the str should be or dict. If str, is it the solver name?

Reply:
I've expanded the docstring.